src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -717,4 +722,41 @@ public static void deleteTemporaryDirectory(Configuration configuration, String
       LOG.debug("Deleted temporary directory '{}'", path);
     }
   }
+
+  public static String generateTimePartitionCondition(StandardTableDefinition tableDefinition,
```

Comment:
It would be nice to add tests for this. Can you log a JIRA for this? No need to do it right now, but it would be good to cover this piece of code with tests.

Reply:
This logic is covered under `generateQuery` Unit test. Have tagged you, PTAL